docs/en/operations/table_engines/odbc.md

```diff
@@ -0,0 +1,29 @@
+# ODBC {#table_engine-odbc}
+
+Allows ClickHouse to connect to external databases via [ODBC](https://en.wikipedia.org/wiki/Open_Database_Connectivity).
+
+To implement ODBC connection, ClickHouse uses the separate program `clickhouse-odbc-bridge`. ClickHouse starts this program automatically when it is required. The ODBC bridge program is installed by the same package with the ClickHouse server.
```

Comment:
* ...as `clickhouse-server`
* needs some explanation about why it's done this way (to safely load ODBC driver for that external database, without risking crashing the main server if the driver is buggy)